docs/user/lib_mapper/ansible.md

```diff
@@ -2,6 +2,7 @@
 | ---------- | -- | ------ |
 | a10.acos_axapi.a10 | → | a10 |
 | arista.eos.eos | → | arista_eos |
+| arubanetworks.aos_switch | → | aruba_aos |
```

Comment:
where is `aruba_aos` originating from?  Its not one of the options from https://github.com/ktbyers/netmiko/blob/develop/PLATFORMS.md which I think is the root of the problem we're having with this one.